tests/e2e/creation_test.go

```diff
@@ -311,6 +305,11 @@ func (tc *testContext) testApplicationCreation(component components.ComponentInt
 			return false, fmt.Errorf("error listing application deployments :%w", err)
 		}
 		if len(appList.Items) != 0 {
```

Comment:
```suggestion
		if len(appList.Items) > 0 {
```
If I'm not wrong, here we are checking item list must have some value i.e. 1,2, etc except 0. 
I think checking greater than zero `0` would be safer than `!=0`.  
wdyt?

Reply:
because you think it can be a negative value?

Reply:
No, Not negative value.  It may be null or undefined etc.

Reply:
> No, Not negative value. It may be null or undefined etc.

https://pkg.go.dev/builtin#len

It's `int`. It cannot be null, undefined etc.